app/src/main/res/values/strings.xml

```diff
@@ -361,4 +361,21 @@
   <string name="topic_revision_recyclerview_tag">topic_revision_recyclerview_tag</string>
   <string name="ongoing_recycler_view_tag">ongoing_recycler_view_tag</string>
   <string name="completed_story_list_recyclerview_tag">completed_story_list_recyclerview_tag</string>
+  <!-- SplashActivity -->
+  <string
+    name="unsupported_app_version_dialog_title"
+    description="The title of the dialog shown when a user's pre-release app has expired.">
+    Unsupported app version
+  </string>
+  <string
+    name="unsupported_app_version_dialog_message"
+    description="The main message of the dialog shown when a user's pre-release app has expired.">
+    This version of the app is no longer supported. Please update it through the Play Store.
+  </string>
+  <string
+    name="unsupported_app_version_dialog_close_button_text"
+    description="The close button text of the dialog shown when a user's pre-release app has expired.">
```

Comment:
Any specific reason behind mentioning this description? Is this related to accessibility?

Reply:
It's related to translations, actually. This is something we should be introducing with all strings since it provides translators context when translating the string contents. This will become a priority later in the year, so I'm starting to create some examples for reference for future PRs.

Reply:
Oh great. This is something new for me. Thanks for explaining.